tools/redpanda-connect/generate-rpcn-connector-docs.js

```
@@ -0,0 +1,202 @@
+        }
+        return item;
+      });
+    } else if (
+      typeof target[key] === 'object' &&
+      typeof overrides[key] === 'object' &&
+      !Array.isArray(target[key]) &&
+      !Array.isArray(overrides[key])
+    ) {
+      // Deep-merge plain objects
+      target[key] = mergeOverrides(target[key], overrides[key]);
+    } else if (['description', 'type'].includes(key) && overrides.hasOwnProperty(key)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)